src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReplacementPlanServiceTests.cs

```diff
@@ -1583,15 +1562,11 @@ bool expectedValidValue
             Assert.Equal(dataSet.Title, replacementPlan.ApiDataSetVersionPlan.DataSetTitle);
             Assert.Equal(dataSetVersion.Id, replacementPlan.ApiDataSetVersionPlan.Id);
             Assert.Equal(dataSetVersion.PublicVersion, replacementPlan.ApiDataSetVersionPlan.Version);
-            if (enableReplacementOfPublicApiDataSets)
-            {
-                Assert.Equal(expectedValidValue, replacementPlan.ApiDataSetVersionPlan.Valid);
-            }
-            else
-            {
-                Assert.Null(replacementPlan.ApiDataSetVersionPlan.MappingStatus);
-                Assert.False(replacementPlan.ApiDataSetVersionPlan.Valid);
-            }
+
+            Assert.NotNull(replacementPlan.ApiDataSetVersionPlan.MappingStatus);
+
+            Assert.Equal(mappingStatus, replacementPlan.ApiDataSetVersionPlan.MappingStatus);
```

Comment:
Minor one, but I think you can remove `Assert.NotNull` here now that you've added `Assert.Equal` because if `replacementPlan.ApiDataSetVersionPlan.MappingStatus` was null, it would not be equal to `mappingStatus` which we know isn't null.